### ISSUE TITLE
[BUG] - Pulling a date from the database

### ISSUE THREAD

**Issue by Auragen-dev** [ws268644/ProjectChocobo#16]
**Describe the bug**
The program appears to have an issue pulling the date from the database

**To Reproduce**
Steps to reproduce the behaviour:
In the branch, go to the events page. It'll bring up this issue on load as I set the datatable in there to load the data when the page is launched

**Expected behaviour**
A clear and concise description of what you expected to happen:
For the datatable to be filled up with the correct information

**Actual behaviour**
A clear and concise description of what actually to happened:
The program couldn't even pull the information from the database.

**Screenshots**
![image](https://user-images.githubusercontent.com/44513246/158252683-83152f33-ce48-4c7d-92ae-b2753064aafb.png)

**Additional context**

**Comment by Auragen-dev** [comment]
I just ran a test. I've got the return working by converting the date into a string. Now I just need to see if I can do the same thing by sending it to the database as a string.
![image](https://user-images.githubusercontent.com/44513246/161266377-f5dbbd98-3480-496d-aec1-d900a846456e.png)
![image](https://user-images.githubusercontent.com/44513246/161266426-d0af3dbc-8979-4467-aa4f-4f7a9e85044a.png)

**Comment by Auragen-dev** [comment]
![image](https://user-images.githubusercontent.com/44513246/161437095-5a828c0f-005f-4c6b-9973-b56e9b6542d2.png)
![image](https://user-images.githubusercontent.com/44513246/161437106-79fcef02-eb2f-4da1-8ec6-67965b27d03b.png)
I tried using ".Text" to get the date selected. The SQL database didn't accept it. I'm going to try using  ".Value" instead

**Comment by Auragen-dev** [comment]
![image](https://user-images.githubusercontent.com/44513246/161437162-2174028e-bd0f-4b68-a74e-15f4deefe1e9.png)
![image](https://user-images.githubusercontent.com/44513246/161437169-aec78762-43ec-45e1-98bc-0c2034eb823b.png)
The SQL server still didn't except the value. I'm going to try some string manipulation to remove the Time entry

**Comment by Auragen-dev** [comment]
![image](https://user-images.githubusercontent.com/44513246/161437353-ab43248b-aa9b-40cf-9d6b-3652a91acbe2.png)
![image](https://user-images.githubusercontent.com/44513246/161437381-06b59d65-1b4f-4caa-9b38-d7da33483e95.png)
So, this time there was data sent to the database. However it came up as blank dates. I think this is because the day and year are in the wrong places. More string manipulation is required

**Comment by Auragen-dev** [comment]
![image](https://user-images.githubusercontent.com/44513246/161437623-5d1cb8c5-b3e0-43c2-9da4-20da35f6660f.png)
![image](https://user-images.githubusercontent.com/44513246/161437631-f6e5868f-d06f-462b-849f-04d4d1aeb996.png)
![image](https://user-images.githubusercontent.com/44513246/161437668-61b92c57-2704-4ada-b77b-e94e6f39abbd.png)

Despite flipping the numbers around the database is still receiving blank dates

**Comment by Auragen-dev** [comment]
![image](https://user-images.githubusercontent.com/44513246/161437773-2c8a3489-0d69-4459-ad4b-1ca371f4b360.png)
![image](https://user-images.githubusercontent.com/44513246/161437787-0b4d4e74-ef6f-4f30-b3a4-4cf0a2410e8b.png)
**I fixed it.**
I wasn't using a stored procedure for this test. I was using a string to send the command to the database. The command was registering the date value as a number, not a string because I didn't have speech marks set in the command itself. This issue shouldn't be an issue as we are solely using stored procedures. But this means I figured out why the database wasn't accepting our dates
